htdocs/core/class/extrafields.class.php

```diff
@@ -1919,15 +1923,15 @@ function getOptionalsFromPost($extralabels,$keyprefix='',$keysuffix='')
 	{
 		global $_POST;
 
-		if (is_array($this->attributes[$object->table_element]['label'])) $extralabels=$this->attributes[$object->table_element]['label'];
```

Comment:
I don't understand this ?
If $object->table_element] does not return what you expect, it probably means you make the call of getOptionalsFromPost on the wrong object.